pkg/config/engine/docker/docker.go

```diff
@@ -128,6 +129,28 @@ func (c Config) Save(path string) (int64, error) {
 		return 0, fmt.Errorf("unable to convert to JSON: %v", err)
 	}
 
-	n, err := engine.Config(path).Write(output)
+	n, err := config.Raw(path).Write(output)
 	return int64(n), err
 }
+
+// GetRuntime returns the TOML object of the runtime passed as input
+func (c *Config) GetRuntime(name string) (interface{}, error) {
```

Comment:
What is the expected behaviour (here and for the other cases) where `name` is not a valid runtime?

Reply:
If the runtime of the `name` does not exist, it should throw an error

Reply:
I've changed this. It'll just return a `nil` when the runime of `name` does not exist